File: src/main/java/com/haozileung/web/domain/system/logging/LoginLog.java
package com.haozileung.web.domain.system.logging;

import java.util.Date;

/**
 * Created by haozi on 16-3-8.
 */
public class LoginLog {
    private Long loginLogId;
    private Long userId;
    private String ipAddress;
    private Date loginTime;
    private Integer result;
    private String browser;
    private String remarks;

    public Long getLoginLogId() {
        return loginLogId;
    }

    public void setLoginLogId(Long loginLogId) {
        this.loginLogId = loginLogId;
    }

    public Long getUserId() {
        return userId;
    }

    public void setUserId(Long userId) {
        this.userId = userId;
    }

    public String getIpAddress() {
        return ipAddress;
    }

    public void setIpAddress(String ipAddress) {
        this.ipAddress = ipAddress;
    }

    public Date getLoginTime() {
        return loginTime;
    }

    public void setLoginTime(Date loginTime) {
        this.loginTime = loginTime;
    }

    public Integer getResult() {
        return result;
    }

    public void setResult(Integer result) {
        this.result = result;
    }

    public String getBrowser() {
        return browser;
    }

    public void setBrowser(String browser) {
        this.browser = browser;
    }

    public String getRemarks() {
        return remarks;
    }

    public void setRemarks(String remarks) {
        this.remarks = remarks;
    }
}

File: src/main/kotlin/com/haozileung/web/domain/logging/LoginLog.kt
/*
 * Powered By [rapid-framework]
 * Web Site: http://www.rapid-framework.org.cn
 * Google Code: http://code.google.com/p/rapid-framework/
 * Since 2008 - 2016
 */

package com.haozileung.web.domain.logging

import java.io.Serializable
import java.util.*

/**
 * @author Haozi
 * *
 * @version 1.0
 * *
 * @since 1.0
 */

data class LoginLog(var loginLogId: Long? = null,
                    var userId: Long? = null,
                    var ipAddress: String? = null,
                    var loginTime: Date? = null,
                    var result: Int? = null,
                    var browser: String? = null,
                    var remarks: String? = null) : Serializable